netci.groovy

```diff
@@ -2212,19 +2218,28 @@ def static calculateBuildCommands(def newJob, def scenario, def branch, def isPR
                         runtestArguments = "${lowerConfiguration} ${arch} ${testOpts}"
 
                         if (doCoreFxTesting) {
-                            def workspaceRelativeFxRoot = "_/fx"
-                            def absoluteFxRoot = "%WORKSPACE%\\_\\fx"
-                            def fxBranch = getFxBranch(branch)
+                            if (scenario == 'corefx_innerloop') {
+                                buildCommands += "tests\\runtest.cmd ${runtestArguments} CoreFXTests"
+                                
+                                // Archive and process (only) the test results
+                                Utilities.addArchival(newJob, "%WORKSPACE%bin/Logs/**/testResults.xml")
+                                Utilities.addXUnitDotNETResults(newJob, "%WORKSPACE%/bin/Logs/**/testResults.xml")
+                            }
+                            else {
+                              def workspaceRelativeFxRoot = "_/fx"
+                              def absoluteFxRoot = "%WORKSPACE%\\_\\fx"
+                              def fxBranch = getFxBranch(branch)
 
-                            buildCommands += "python -u %WORKSPACE%\\tests\\scripts\\run-corefx-tests.py -arch ${arch} -ci_arch ${architecture} -build_type ${configuration} -fx_root ${absoluteFxRoot} -fx_branch ${fxBranch} -env_script ${envScriptPath}"
+                              buildCommands += "python -u %WORKSPACE%\\tests\\scripts\\run-corefx-tests.py -arch ${arch} -ci_arch ${architecture} -build_type ${configuration} -fx_root ${absoluteFxRoot} -fx_branch ${fxBranch} -env_script ${envScriptPath}"
```

Comment:
What is the difference between the outerloop and innerloop jobs? Why are we opting for two different ways to run corefx tests?

Reply:
CoreFx tests have a notion of innerloop and outerloop built into them as well.  We should probably only run corefx's innerloop tests in coreclr's innerloop.  This naming seems appropriate in case we want to add corefx outerloop jobs at some point.  It's unclear right now (to me, at least).

---
In reply to: [196827178](https://github.com/dotnet/coreclr/pull/18388#discussion_r196827178) [](ancestors = 196827178)

Reply:
Yep, @RussKeldorph was spot on. The CI configuration will only run innerloop (marked as such) CoreFX tests during innerloop CoreCLR jobs.